### ISSUE TITLE
Auto-upload awaiting for wifi if conflict on server

### ISSUE THREAD

**Issue by Shagequi** [nextcloud/android#6080]
Hi,

There is a bug in recent versions (master, 3.12) during auto-upload.

Here are the steps : 
1) Put some pictures of your phone "Camera" in a new folder on the server (the same folder that'll be synchronized in case of an auto-upload)
2) Enable the auto-upload, and check "Also upload existing files", and uncheck "Only upload on unmetered Wi-Fi". As remote folder, use the folder you created for the step 1, save.
3) Restart the app (if wanted)
4) Files will wait for wifi and/or the conflict management will not be functionnal in this case, the auto-upload will not work for existing files but you can't understand why (it's not explained).

![Capture d’écran 2020-05-15 à 10 26 27](https://user-images.githubusercontent.com/7050479/82028961-9195dd00-9696-11ea-84c4-47af5e9b6daf.png)

**Comment by offlinehoster** [comment]
I see the same issue.
Pictures not uploaded yet. Still waiting for "WIFI".

**Comment by offlinehoster** [comment]
I just turned off the "upload only by wifi" button, it uploads now a newly taken picture...but the already existing, new pictures are not uploading yet.

**Comment by offlinehoster** [comment]
Uploads by mobil data only are working.

**Comment by Shagequi** [comment]
@offlinehoster The solution is to delete the files already on the server, or to refuse the upload of files in conflict

**Comment by offlinehoster** [comment]
@Shagequi no..thats not working here. I just deleted the file inside the waiting upload folder on the nextcloud app....now the app recognized the "new" picture as not uploaded yet and it's still waiting for a wifi connection.

**Comment by tobiasKaminsky** [comment]
@offlinehoster which version do you use?

**Comment by tobiasKaminsky** [comment]
I think both is solved via #6107

**Comment by offlinehoster** [comment]
What's next? Still got that issue...
I'm using Version 20200517 from Fdroid.

Nextcloud is running with 18.0.5  

so it looks like that my system is up2date?!

**Comment by AndyScherzinger** [comment]
@offlinehoster the fix @tobiasKaminsky is refering to https://github.com/nextcloud/android/commit/f0da2f41e1741718d7eabbe8a3630a3f65a1a8ea which is available in the dev build starting with dev-20200521 as can be seen at the top of the page of the link I added so your client is not having that fix. So your system is far from up2date ;) Other than that the latest dev build is https://github.com/nextcloud/android/releases/tag/dev-20200611

**Comment by ilvalle** [comment]
@AndyScherzinger is there any plan to have https://github.com/nextcloud/android/commit/f0da2f41e1741718d7eabbe8a3630a3f65a1a8ea included in the next stable release?

**Comment by AndyScherzinger** [comment]
@ilvalle as linked in the PR it'll ship with 3.13 which is the next scheduled stable release, so yes😀

**Comment by ilvalle** [comment]
@AndyScherzinger right. I asked because is not listed in https://github.com/nextcloud/android/issues?q=is%3Aopen+is%3Aissue+milestone%3A%22Nextcloud+App+3.13.0%22

**Comment by AndyScherzinger** [comment]
Actually it is. The commit you are referring to is part of  #6107 which is assigned to 3.13.0 😉
